string offset_enp45s0_systemclock
string offset_enp45s0_enp46s0
string offset_mgbe0_enp45s0
string status_enp46s0_ptp4l

# frequencies names have to be the topic names, where the '/' are replaced with '_' and "_hz" is appended
float64 gt_box_adis16475_hz
float64 gt_box_usb_cam_image_hz